### PR TITLE
Feature/microservice

### PR DESCRIPTION
**Pull request by Abezzam10**
This is a basic microservice pattern that handles last seen, user info and calculations.
Returns JSON for ease of API use
Developing RESTful services working with JSON

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    # Enable protection agains *Cross-site Request Forgery (CSRF)*
+    WTF_CSRF_ENABLED = True
+
+    # Use a secure, unique and absolutely secret key for
+    # signing the data.
+    CSRF_SESSION_KEY = os.getenv('CSRF_SESSION_KEY')
+
+    # Secret key for signing cookies
+    if os.environ.get('SECRET_KEY'):
+        SECRET_KEY = os.environ.get('SECRET_KEY')
```
Comment (CCorrado) [inline config.py]: ooooo environment variables -- fancy :)

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
+alembic==0.8.10
```
Comment (CCorrado) [inline requirements.txt]: nice use of requirements

**Comment by CCorrado** [comment]
Fixes #2

**Comment by Abezzam10** [comment]
> check out the README to see how to view the Swagger docs

I'm somehow not able to view it

**Comment by CCorrado** [comment]
screen shot 2019-03-03 at 8 51 20 pm

weird...it definitely is working/passing on travis as well.

**Comment by CCorrado** [comment]
@Rosovski  @SamNormcoreWayne  any other thoughts here? Want to get this in so we can update our progress for this week.

**Comment by CCorrado** [comment]
@Rosovski @SamNormcoreWayne  bump

**Pull request merged**